### PR TITLE
D14 partI done

### PR DESCRIPTION
**Pull request by taherkheli**

### CODE REVIEW

File: src/D14/Emulator.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    public long PartI(List<Tuple<string, long, long>> list) 
+    {
+      string mask;
+      long address;
+      long value;
+      long sum = 0;
+
+      foreach (var item in list)
+      {
+        mask = item.Item1;
```
Comment (Jaxwood) [inline src/D14/Emulator.cs]: @taherkheli I think this can also be written as `var (mask, address, value) = item;`

Comment (Jaxwood) [reply src/D14/Emulator.cs]: or you can put the destructoring in the foreach declaration

Comment (Jaxwood) [reply src/D14/Emulator.cs]: see more [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples)

Comment (taherkheli) [reply src/D14/Emulator.cs]: all great but you should have written `(mask, address, value) = item;` instead ;) .. fixed

**Pull request merged**